src/main/resources/META-INF/plugin.xml

```diff
@@ -10,6 +10,19 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>0.7.0</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/214">As a user I want to be able to open a terminal in a pod</a></li>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/461">cannot follow logs for a pod with multiple containers</a></li>
```

Comment:
Maybe to have the first letter as capital as the other points

Reply:
This is the changelog. I take the exact wording of the issue and copy it over here, 1:1. I see no point in correcting this here.

Reply:
This text is the one that users see in the marketplace https://plugins.jetbrains.com/plugin/15921-kubernetes-by-red-hat . Up to you what to do but for me it should be taken care a bit. It's like when you write a text and you don't pay attention with punctuation. It's ok if you're in a chat, a bit less for something formal. 
I also saw Mohit comments for the wording. I suck at english so i cannot give any opinion on it but if there is a way to explain better what's the improvement, then it's a change to make for me.  Sticking with the idea that it has to be 1:1 as the changelog is counterproductive from my pov. This is your main ads.
Just my 2 cent. :slightly_smiling_face: